### ISSUE TITLE
Running Autotests Failing

### ISSUE THREAD

**Issue by amjadraza** [embedchain/embedchain#283]
I am trying to run the tests and have the below error. 

version: recently updated 0.0.23

__________________________________________________________________________________ TestApp.test_query __________________________________________________________________________________

self = <test_query.TestApp testMethod=test_query>

    def setUp(self):
>       self.app = App()

tests/embedchain/test_query.py:13:
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _
embedchain/embedchain.py:329: in __init__
    config = InitConfig()
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _

self = <embedchain.config.InitConfig.InitConfig object at 0x7eff450f7190>, log_level = None, ef = None, db = None, host = None, port = None, id = None

    def __init__(self, log_level=None, ef=None, db=None, host=None, port=None, id=None):
        """
        :param log_level: Optional. (String) Debug level
        ['DEBUG', 'INFO', 'WARNING', 'ERROR', 'CRITICAL'].
        :param ef: Optional. Embedding function to use.
        :param db: Optional. (Vector) database to use for embeddings.
        :param id: Optional. ID of the app. Document metadata will have this id.
        :param host: Optional. Hostname for the database server.
        :param port: Optional. Port for the database server.
        """
        self._setup_logging(log_level)

        if db is None:
            from embedchain.vectordb.chroma_db import ChromaDB

>           self.db = ChromaDB(ef=self.ef)
E           AttributeError: 'InitConfig' object has no attribute 'ef'

embedchain/config/InitConfig.py:27: AttributeError
__________________________________________________________________ TestChromaDbHostsInit.test_init_with_host_and_port __________________________________________________________________

**Comment by amjadraza** [comment]
Solved by `self.db = ChromaDB(ef=ef) ` instead of self.db = ChromaDB(ef=self.ef)

```
self._setup_logging(log_level)

        if db is None:
            from embedchain.vectordb.chroma_db import ChromaDB

            self.db = ChromaDB(ef=ef)
        else:
            self.db = db

        self.ef = ef
        self.host = host
        self.port = port
        self.id = id
```

**Comment by amjadraza** [comment]
@deshraj did you check this too?

**Comment by amjadraza** [comment]
Fixed with PR #281
